nextstrain/cli/authn.py

```diff
@@ -48,7 +48,7 @@ def __init__(self, session: cognito.Session):
         assert session.id_claims
 
         self.username = session.id_claims["cognito:username"]
-        self.groups   = session.id_claims["cognito:groups"]
+        self.groups   = session.id_claims.get("cognito:groups", "")
```

Comment:
```suggestion
        self.groups   = session.id_claims.get("cognito:groups", [])
```

The type of `self.groups` is `List[str]`

https://github.com/nextstrain/cli/blob/1247a9be07939bcbf59394371112af1f1706adf4/nextstrain/cli/authn.py#L38-L45

as the `cognito:groups` claim is a JSON array of strings.

Reply:
Not sure why the type checking didn't catch that, but it's not entirely surprising as the type checking isn't comprehensive, just "best effort".

Reply:
Thanks, good catch! [Fixed](https://github.com/nextstrain/cli/compare/1ccbed2bf6624e41cc2399920fbd789cf856dfd4..9cbb8042874f60dafe812527d923d13664782b21).

I think it's because `session.id_claims` has an implicit type of `Any`, so type checking is [ignored](https://mypy.readthedocs.io/en/stable/common_issues.html#no-errors-reported-for-obviously-wrong-code) on that entire expression.